pallets/liquidity-pools-gateway/routers/Cargo.toml

```diff
@@ -43,23 +43,24 @@ pallet-liquidity-pools-gateway = { path = "../.", default-features = false }
 [dev-dependencies]
 lazy_static = "1.4.0"
 
-cumulus-primitives-core = { git = "https://github.com/moonbeam-foundation/cumulus", default-features = false, branch = "moonbeam-polkadot-v0.9.43" }
+cumulus-primitives-core = { workspace = true }
 
-xcm-builder = { git = "https://github.com/moonbeam-foundation/polkadot", default-features = false, branch = "moonbeam-polkadot-v0.9.43" }
-xcm-executor = { git = "https://github.com/moonbeam-foundation/polkadot", default-features = false, branch = "moonbeam-polkadot-v0.9.43" }
+xcm-builder = { workspace = true }
+xcm-executor = { workspace = true }
 
-pallet-evm-chain-id = { git = "https://github.com/moonbeam-foundation/frontier", default-features = false, branch = "moonbeam-polkadot-v0.9.43" }
-pallet-evm-precompile-simple = { git = "https://github.com/moonbeam-foundation/frontier", default-features = false, branch = "moonbeam-polkadot-v0.9.43" }
-pallet-timestamp = { git = "https://github.com/moonbeam-foundation/substrate", branch = "moonbeam-polkadot-v0.9.43" }
+pallet-evm-chain-id = { workspace = true }
+pallet-evm-precompile-simple = { workspace = true }
+pallet-timestamp = { workspace = true, default-features = true }
 
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
-sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
+sp-core = { workspace = true }
+sp-io = { workspace = true }
 
-orml-traits = { git = "https://github.com/moonbeam-foundation/open-runtime-module-library", branch = "moonbeam-polkadot-v0.9.43", default-features = false }
+# TODO(william): Used to be "https://github.com/moonbeam-foundation/open-runtime-module-library", branch = "moonbeam-polkadot-v0.9.43"
```

Comment:
Nit - do we still need this `TODO`?

Reply:
Actually, just realising that I skipped to double-check this pallet crate because it was under `gateway` and it still contains paths.

Reply:
But they're local so not big issue